README.md

```diff
@@ -1,3 +1,5 @@
+<img width="200" alt="Tag-runtime" src="https://user-images.githubusercontent.com/7659560/160054856-5b05fa62-c5f0-4483-b74d-97e080fe6fd7.png">
```

Comment:
Maybe better to put image into this repo and reference it directly within one repo?

Reply:
Not a fan of storing images in a repo, but any suggestions about what directory to store these under?

Reply:
oh, that works!